salt/netapi/rest_tornado/saltnado.py

```diff
@@ -250,10 +250,10 @@ def saltclients(self):
             local_client = salt.client.get_local_client(mopts=self.application.opts)
             # TODO: refreshing clients using cachedict
             SaltClientsMixIn.__saltclients = {
-                'local': local_client.run_job,
+                'local': local_client.run_job_async,
```

Comment:
The 'local' is a wrapper for `salt cmd.run ...`, and the 'local_async' is a wrapper for `salt --async cmd.run ...`.
They both need to trigger job, thus they have triggered salt job using `run_job`. But the `run_job` is not coroutine function, so I changed it into the coroutine function `run_job_async`.